guides/searcher-onboarding.md

```diff
@@ -177,16 +230,16 @@ The signature needs to be provided via the 'X-Flashbots-Signature' Header. Refer
 ### Why is there rate limiting?
 Rate limiting is currently in place to protect the infrastructure sitting behind it. Similar to how Infura has a gas limit on eth_call.
 
-
 ### Why do I need to provide this unrelated auth-signing key?
-
 By signing payloads with your own relay signing key, this will enable building a reputation for high-priority delivery of your bundles to miners. MEV-relay simulates bundles before sending to miners which can take a small amount of time. MEV-relay cannot determine which bundles are profitable without performing a full simulation. This signing key allows the relay to infer which bundles are likely profitable, based on historical performance. Using a reputation system allows reliable searchers to be rewarded while still allowing new searchers to participate.
 
+### Are you on any testnets?
+Yes, we are on Goerli. Our endpoint is `https://relay-goerli.flashbots.net/`.
```

Comment:
I need to update ethers library to show how to do this. It's a little tricky, because you will need to get a goerli real endpoint as well, for nonce/genericProvider